runtime/themes/ayu_evolve.toml

```
@@ -0,0 +1,36 @@
+inherits = 'ayu_dark'
+
+"keyword.control" = "orange"
+"keyword.storage" = "yellow"
+"keyword.storage.modifier" = "magenta"
+"variable.other.member" = "#gray"
```

```suggestion
"variable.other.member" = "gray"
```

to use the palette color